js/widgets/musickeyboard.js

```diff
@@ -710,7 +816,7 @@ function MusicKeyboard() {
                 sortableList.push({
                     frequency: noteToFrequency(
                         this.noteNames[i] + this.octaves[i],
-                        this._logo.turtles.ithTurtle(0).singer.keySignature
+                        logo.turtles.ithTurtle(0).singer.keySignature
```

Comment:
I think `turtles` is a global too. We can directly do `turtles.ithTurtle` no need to access it through `logo`.

Reply:
Thanks for pointing out, committing the changes asap